gpu.Dockerfile

```diff
@@ -56,7 +56,6 @@ RUN apt-get install -y --no-install-recommends ocl-icd-opencl-dev && \
 # Install GPU specific packages
 RUN CPATH=/usr/local/cuda/targets/x86_64-linux/include install2.r --error --ncpus $ncpus --repo http://cran.rstudio.com \
     h2o4gpu \
-    bayesCL \
```

Comment:
Any reason we're removing this package?

Reply:
See the previous comment and link to discussion.